homeassistant/components/icloud/device_tracker.py

```diff
@@ -49,12 +53,12 @@ def __init__(self, device: IcloudDevice):
         self._unsub_dispatcher = None
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return a unique ID."""
-        return f"{self._device.unique_id}_tracker"
+        return self._device.unique_id
```

Comment:
The `unique_id` is unique by platform right ?

Or should I add `_tracker` again ?

Reply:
Yes, it's enough that the id is unique per platform.

Have we released the integration with the current device tracker unique_id? In that case it's good to avoid a breaking change by not changing unique_id.

Reply:
It’s not released, the last PR was merge just after 0.103.0 released, so for next 0.104.0